php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -651,6 +652,8 @@ public static void expression(LexerlessGrammarBuilder b) {
   }
 
   public static void declaration(LexerlessGrammarBuilder b) {
+    // TODO: use declaration should be simplified and merged with use_statement
+    // FIXME: also USE_CONST_DECLARATION_STATEMENT is never used...
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)